### PR TITLE
Add pod file and swap UI docker file

### PR DESCRIPTION
**Pull request by nlordell**
This PR introduces a pod file for starting up an entire GPv2 system (including CowSwap) for local debugging.

This PR is still a work in progress, just wanted to share what I have so far.

### Test Plan

TBD

### CODE REVIEW

**Comment by vkgnosis** [comment]
Ohh even with the UI. That is cool.

File: docker/debug/swapui.env [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+REACT_APP_NETWORK_URL_100=https://rpc.xdaichain.com
+REACT_APP_NETWORK_URL_1337=http://localhost:8545
+
+# Domain
+REACT_APP_DOMAIN_REGEX_DEV=".*"
+
+# API
+REACT_APP_API_URL_STAGING_MAINNET=http://localhost:8080/api
+REACT_APP_API_URL_STAGING_RINKEBY=http://localhost:8080/api
+REACT_APP_API_URL_STAGING_XDAI=http://localhost:8080/api
```
Comment (fleupold) [inline docker/debug/swapui.env:18]: Does this need one for 1337 as well?

File: docker/debug/pod.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+spec:
+  containers:
+    - name: solver
+      image: gnosispm/gp-v2-services
+      args: ["solver"]
+      env:
+        - name: PRIVATE_KEY
+          value: "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+        - name: AMM_SOURCES
+          value: Uniswap
```
Comment (fleupold) [inline docker/debug/pod.yaml:15]: Why that restriction? Because sushi swap is not yet deployed?

**Review by fleupold — COMMENTED** [review]
Can you add a section to the readme how to use this? I think we commonly want to run at least one service on the host system so that we can set break points and change code without having to rebuild the container.

File: docker/debug/pod.yaml (hunk 2 above)
Comment (nlordell) [reply docker/debug/pod.yaml:15]: We don't deploy SushiSwap on Ganache. (with `cargo run --bin deploy`).

File: docker/debug/swapui.env (hunk 1 above)
Comment (nlordell) [reply docker/debug/swapui.env:18]: Good question. I'll dig around.

**Comment by nlordell** [comment]
> Can you add a section to the readme how to use this?

For sure!

**Pull request closed without merging**